src/qml/components/AboutOptions.qml

```diff
@@ -13,42 +13,56 @@ ColumnLayout {
         Layout.fillWidth: true
         header: qsTr("Website")
         actionItem: ExternalLink {
-            description: qsTr("bitcoincore.org >")
+            description: qsTr("bitcoincore.org")
             link: "https://bitcoincore.org"
+            iconSource: "image://images/caret-right"
+            iconWidth: 18
+            iconHeight: 18
         }
     }
     Setting {
         Layout.fillWidth: true
         header: qsTr("Source code")
         actionItem: ExternalLink {
-            description: qsTr("github.com/bitcoin/bitcoin >")
+            description: qsTr("github.com/bitcoin/bitcoin")
             link: "https://github.com/bitcoin/bitcoin"
+            iconSource: "image://images/caret-right"
+            iconWidth: 18
+            iconHeight: 18
         }
     }
     Setting {
         Layout.fillWidth: true
         header: qsTr("License")
         actionItem: ExternalLink {
-            description: qsTr("MIT >")
+            description: qsTr("MIT")
             link: "https://opensource.org/licenses/MIT"
+            iconSource: "image://images/caret-right"
+            iconWidth: 18
+            iconHeight: 18
         }
     }
     Setting {
         Layout.fillWidth: true
         header: qsTr("Version")
         actionItem: ExternalLink {
-            description: qsTr("v22.99.0-1e7564eca8a6 >")
+            description: qsTr("v22.99.0-1e7564eca8a6")
             link: "https://bitcoin.org/en/download"
+            iconSource: "image://images/caret-right"
+            iconWidth: 18
+            iconHeight: 18
         }
     }
     Setting {
         Layout.fillWidth: true
         header: qsTr("Developer options")
         description: qsTr("Only use these if you have development experience")
-        actionItem: TextButton {
-            text: ">"
-            bold: false
-            rightalign: true
+        actionItem: Button {
+            icon.source: "image://images/caret-right"
+            icon.color: Theme.color.neutral9
+            icon.height: 18
+            icon.width: 18
+            background: null
```

Comment:
Why are the `icon.color` and `background` properties are specified for some elements, but not for others?

Reply:
Button is a qml type, so we need to specify the `icon.color`, background value explicitly for each time the Button is used as `actionItem`, which this PR does.

Whereas, the `ExternalLink` is a custom-defined component that already employs the respective icon.color, and background value in its definition, so we need not specify that separately when using it.

In `src/qml/controls/ExternalLink.qml`

```qml
            sourceComponent: Button {
                icon.source: root.iconSource
                icon.color: Theme.color.neutral9
                icon.height: root.iconHeight
                icon.width: root.iconWidth
                background: null
                onClicked: Qt.openUrlExternally(link)
            }
```